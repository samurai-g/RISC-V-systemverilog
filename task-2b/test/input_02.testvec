A
8a726f07
fd62bd44
942d99fd
29397e51
7d731be2
1c815f6b
e4687e8e
fbd814b0
5106a70f
e4b8ad8d
