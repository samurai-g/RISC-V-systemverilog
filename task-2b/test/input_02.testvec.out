8a726f07
942d99fd
e4687e8e
e4b8ad8d
fbd814b0
fd62bd44
1c815f6b
29397e51
5106a70f
7d731be2
